web/index.html

```diff
@@ -447,9 +447,9 @@ <h4 class="card-title">Interessen</h4>
                                                     <i class="far fa-comments"></i>
                                                 </div>
                                                 <div class="card-body">
-                                                    <h4 class="card-title">Charakter</h4>
+                                                    <h4 class="card-title">Lebensansichten</h4>
```

Comment:
Über das mit den Lebensansichten möchte ich mit dir nochmal sprechen. Die Idee finde ich gut, nur steht dazu aktuell noch kein Konzept. Und erst wenn wir ein Konzept haben, würde ich es in die Website mit einbauen.